processor/redactionprocessor/testdata/config.yaml

```diff
@@ -16,10 +16,21 @@ processors:
       - group
       - id
       - name
-    # Blocklist for span attribute values
+    # BlockedValues is a list of regular expressions for blocking values of
+    # allowed span attributes. Values that match are masked
     blocked_values:
       - "4[0-9]{12}(?:[0-9]{3})?" ## Visa credit card number
       - "(5[1-5][0-9]{14})"       ## MasterCard number
+    # DryRun mode documents the changes the processor would make without
+    # deleting your data. You can use it to confirm that the right span
+    # attributes will be redacted
+    dry_run: false
```

Comment:
This can be achieved by having a "separate" pipeline with only logging/debugging exporter. We don't have this pattern anywhere else because of that.

Reply:
@bogdandrutu That makes sense. I've now removed the dry run mode from the PR